src/akushon/action/model/action.cpp

```diff
@@ -110,4 +110,48 @@ void Action::reset()
   poses.clear();
 }
 
+void Action::map_action(const Action & source_action, const Action & target_action, int target_pose_index, float source_val, float source_min, float source_max)
+{
+  std::vector<tachimawari::joint::Joint> src_joints = source_action.get_pose(target_pose_index).get_joints();
+  std::vector<tachimawari::joint::Joint> target_joints = target_action.get_pose(target_pose_index).get_joints();
+  for (int joint = 0; joint < int(src_joints.size()); joint++)
```

Comment:
Prefer static casting over other kind of casting:
```suggestion
  for (int joint = 0; joint < static_cast<int>(src_joints.size()); joint++)
```